Source/WebKit/UIProcess/Extensions/WebExtensionController.h

```diff
@@ -108,10 +108,9 @@ class WebExtensionController : public API::ObjectImpl<API::Object::Type::WebExte
     void getDataRecords(OptionSet<WebExtensionDataType>, CompletionHandler<void(Vector<Ref<WebExtensionDataRecord>>)>&&);
     void getDataRecord(OptionSet<WebExtensionDataType>, WebExtensionContext&, CompletionHandler<void(RefPtr<WebExtensionDataRecord>)>&&);
     void removeData(OptionSet<WebExtensionDataType>, const Vector<Ref<WebExtensionDataRecord>>&, CompletionHandler<void()>&&);
-    void removeData(OptionSet<WebExtensionDataType>, const WebExtensionContextSet&, CompletionHandler<void()>&&);
 
-    void calculateStorageSize(_WKWebExtensionStorageSQLiteStore *, WebExtensionDataType, CompletionHandler<void(size_t)>&&);
-    void removeStorage(_WKWebExtensionStorageSQLiteStore *, WebExtensionDataType, CompletionHandler<void()>&&);
+    void calculateStorageSize(_WKWebExtensionStorageSQLiteStore *, WebExtensionDataType, CompletionHandler<void(size_t, NSString *errorMessage)>&&);
```

Comment:
This should probably be `CompletionHandler<void(Expected<size_t, WebExtensionError>&&)>&&` if size_t and error message cannot appear at the same time.

Reply:
`getStorageSizeForKeys` will actually return 0 if there was an error calculating the storage size, which would be passed in here

Reply:
But 0 can be assumed with the error, so `Expected<size_t, WebExtensionError>` would be best here.

Reply:
but i guess i could pass in an Unexpected error into the completion handler here instead